### PR TITLE
Use ShapeOf to create a new shape for gamma, mean, beta in BatchNormD…

### PR DESCRIPTION
**Pull request by mateusztabaka**
…ecomposition

### CODE REVIEW

File: inference-engine/src/transformations/src/transformations/op_conversions/batch_norm_decomposition.cpp [hunk 1]
```diff
@@ -58,12 +58,13 @@ ngraph::pass::BatchNormDecomposition::BatchNormDecomposition() {
         auto gamma_div_scale = std::make_shared<opset5::Divide>(m_gamma, scale);
 
         int64_t dims_to_add = m_input.get_partial_shape().rank().get_length() - 2;
-
-        // TODO: instead of getting full shape we can concatenate sequence of ones with ShapeOf
-        Shape input_aligned_shape = m_gamma.get_shape();
-        for (int64_t i = 0; i < dims_to_add; ++i)
-            input_aligned_shape.push_back(1);
-        auto new_shape = opset5::Constant::create(element::i64, Shape{input_aligned_shape.size()}, input_aligned_shape);
+        const auto one = op::Constant::create(element::i64, Shape{1}, {1});
```
Comment (ilyachur) [inline inference-engine/src/transformations/src/transformations/op_conversions/batch_norm_decomposition.cpp:61]: Should we add new tests to cover the case?

Comment (mateusztabaka) [reply inference-engine/src/transformations/src/transformations/op_conversions/batch_norm_decomposition.cpp:61]: BatchNorm single layer tests cover the case

**Review by ilyachur — APPROVED** [review]
